packages/bierzo-wallet/src/components/Header/components/HiMenu/index.tsx

```diff
@@ -71,8 +75,14 @@ const onPolicy = (): void => {
 
 const HiMenu = (props: {}): JSX.Element => {
   const classes = useStyles();
+  const dispatch = ReactRedux.useDispatch();
   const { ...rest } = props;
 
+  const onLogout = (): void => {
+    dispatch(resetAppAction());
```

Comment:
Does this also change the location?

Reply:
Yes, It will change location also https://github.com/iov-one/ponferrada/blob/a457876b8d008007a0acdcee8cfb9b235d6dda37/packages/bierzo-wallet/src/components/RequireLogin/index.tsx#L19

Reply:
Thanks for checking!